[itk] Build failue

Package: itk[core,python]:arm64-osx -> 5.2.1#5

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 03a211eb3 2023-04-20 (24 hours ago)

**To Reproduce**

`vcpkg install --enforce-port-checks --allow-unsupported`

**Failure logs**

```
-- Using cached InsightSoftwareConsortium-ITK-95800fd4d4b08678a6c0ebb63eb242893025b660.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/InsightSoftwareConsortium-ITK-95800fd4d4b08678a6c0ebb63eb242893025b660.tar.gz
-- Applying patch double-conversion.patch
-- Applying patch openjpeg.patch
-- Applying patch openjpeg2.patch
-- Applying patch var_libraries.patch
-- Applying patch wrapping.patch
-- Applying patch opencl.patch
-- Applying patch use-the-lrintf-intrinsic.patch
-- Applying patch dont-build-gtest.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean
-- itk builds a long time (>1h) with python wrappers enabled!
-- Found external ninja('1.11.1').
-- Configuring arm64-osx-dbg
-- Configuring arm64-osx-rel
CMake Warning at vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:314 (message):
  The following variables are not used in CMakeLists.txt:

      BUILD_PKGCONFIG_FILES
      DCMTK_USE_ICU
      INSTALL_GTEST
      ITK_USE_SYSTEM_GOOGLETEST
      ITK_USE_SYSTEM_ICU
      Module_ITKCudaCommon
      RTK_BUILD_APPLICATIONS

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/itk/portfile.cmake:126 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)


-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/itk/portfile.cmake:190 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/install-arm64-osx-dbg-out.log</summary>

```
[1/3844] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DKWSYS_NAMESPACE=itksys -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Modules/ThirdParty/KWSys/src -fPIC  -Wno-uninitialized -Wno-unused-parameter -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow  -w -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/Base64.c.o -MF Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/Base64.c.o.d -o Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/Base64.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/ThirdParty/KWSys/src/KWSys/Base64.c
[2/3844] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DKWSYS_NAMESPACE=itksys -DKWSYS_ENCODING_DEFAULT_CODEPAGE=CP_UTF8 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Modules/ThirdParty/KWSys/src -fPIC  -Wno-uninitialized -Wno-unused-parameter -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow  -w -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/EncodingC.c.o -MF Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/EncodingC.c.o.d -o Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/EncodingC.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/ThirdParty/KWSys/src/KWSys/EncodingC.c
[3/3844] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DKWSYS_NAMESPACE=itksys -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Modules/ThirdParty/KWSys/src -fPIC  -Wno-uninitialized -Wno-unused-parameter -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow  -w -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -MD -MT Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/MD5.c.o -MF Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/MD5.c.o.d -o Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/MD5.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/ThirdParty/KWSys/src/KWSys/MD5.c
[4/3844] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DKWSYS_NAMESPACE=itksys -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Modules/ThirdParty/KWSys/src -fPIC  -Wno-uninitialized -Wno-unused-parameter -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow  -w -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -DKWSYS_C_HAS_PTRDIFF_T=1 -DKWSYS_C_HAS_SSIZE_T=1 -DKWSYS_C_HAS_CLOCK_GETTIME_MONOTONIC=1 -MD -MT Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/System.c.o -MF Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/System.c.o.d -o Modules/ThirdParty/KWSys/src/KWSys/CMakeFiles/itksys.dir/System.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/ThirdParty/KWSys/src/KWSys/System.c
...
Skipped 1277 lines
...

[1155/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray.cxx
FAILED: Wrapping/itkArray.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1156/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBackwardDifferenceOperator.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBackwardDifferenceOperator.cxx
FAILED: Wrapping/itkBackwardDifferenceOperator.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBackwardDifferenceOperator.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBackwardDifferenceOperator.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBackwardDifferenceOperator.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBackwardDifferenceOperator.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1157/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkCovariantVector.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkCovariantVector.cxx
FAILED: Wrapping/itkCovariantVector.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkCovariantVector.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkCovariantVector.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkCovariantVector.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkCovariantVector.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1158/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBSplineInterpolationWeightFunction.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBSplineInterpolationWeightFunction.cxx
FAILED: Wrapping/itkBSplineInterpolationWeightFunction.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBSplineInterpolationWeightFunction.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBSplineInterpolationWeightFunction.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBSplineInterpolationWeightFunction.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBSplineInterpolationWeightFunction.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1159/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkAnnulusOperator.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkAnnulusOperator.cxx
FAILED: Wrapping/itkAnnulusOperator.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkAnnulusOperator.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkAnnulusOperator.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkAnnulusOperator.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkAnnulusOperator.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1160/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray2D.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray2D.cxx
FAILED: Wrapping/itkArray2D.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray2D.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray2D.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray2D.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkArray2D.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1161/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkContinuousIndex.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkContinuousIndex.cxx
FAILED: Wrapping/itkContinuousIndex.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkContinuousIndex.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkContinuousIndex.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkContinuousIndex.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkContinuousIndex.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1162/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultStaticMeshTraits.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultStaticMeshTraits.cxx
FAILED: Wrapping/itkDefaultStaticMeshTraits.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultStaticMeshTraits.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultStaticMeshTraits.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultStaticMeshTraits.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultStaticMeshTraits.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1163/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultDynamicMeshTraits.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultDynamicMeshTraits.cxx
FAILED: Wrapping/itkDefaultDynamicMeshTraits.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultDynamicMeshTraits.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultDynamicMeshTraits.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultDynamicMeshTraits.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkDefaultDynamicMeshTraits.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1164/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBoundingBox.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBoundingBox.cxx
FAILED: Wrapping/itkBoundingBox.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBoundingBox.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBoundingBox.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBoundingBox.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/itkBoundingBox.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
[1165/3844] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommonBase.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommonBase.cxx
FAILED: Wrapping/ITKCommonBase.xml /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommonBase.xml 
cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Modules/ITKCommon && env SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk MACOSX_DEPLOYMENT_TARGET= /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/Generators/CastXML/castxml/bin/castxml -o /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommonBase.xml --castxml-gccxml --castxml-start _wrapping_ --castxml-cc-gnu "(" /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -Wall -Wcast-align -Wdisabled-optimization -Wextra -Wformat=2 -Winvalid-pch -Wno-format-nonliteral -Wpointer-arith -Wshadow -Wunused -Wwrite-strings -Wno-strict-overflow -Wno-deprecated -Wno-invalid-offsetof -Wno-undefined-var-template -Woverloaded-virtual -std=c++14 ")" -w -c @/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommon.castxml.inc /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommonBase.cxx
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/arm64-osx-dbg/Wrapping/ITKCommonBase.cxx:1:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkCommand.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkObject.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkLightObject.h:21:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/itk/src/893025b660-829cab57a0.clean/Modules/Core/Common/include/itkMacro.h:45:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/typeinfo:64:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/exception:84:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:135:9: error: no member named 'at_quick_exit' in the global namespace
using ::at_quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/cstdlib:136:9: error: no member named 'quick_exit' in the global namespace
using ::quick_exit _LIBCPP_USING_IF_EXISTS;
      ~~^
2 errors generated.
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "itk",
      "default-features": true,
      "features": [
        "python"
      ]
    }
  ],
  "$overrides": [
    {
      "name": "capstone",
      "version": "4.0.1-120373dc"
    }
  ],
  "$builtin-baseline": "338f99eed499b6df1628829e40695b396a70a1f7"
}

```
</details>

--- a/ports/itk/cxx17.patch
+++ b/ports/itk/cxx17.patch
@@ -1,0 +1,12 @@
+diff --git a/CMake/ITKInitializeCXXStandard.cmake b/CMake/ITKInitializeCXXStandard.cmake
+index 5345230b..a9ca48c3 100644
+--- a/CMake/ITKInitializeCXXStandard.cmake
++++ b/CMake/ITKInitializeCXXStandard.cmake
+@@ -1,6 +1,6 @@
+ ##  Set the default target properties for ITK
+ if(NOT CMAKE_CXX_STANDARD)
+-  set(CMAKE_CXX_STANDARD 14) # Supported values are 14, 17, 20, and 23.
++  set(CMAKE_CXX_STANDARD 17) # Supported values are 14, 17, 20, and 23.
+ endif()
+ if(NOT CMAKE_CXX_STANDARD_REQUIRED)
+   set(CMAKE_CXX_STANDARD_REQUIRED ON)

--- a/ports/itk/portfile.cmake
+++ b/ports/itk/portfile.cmake
@@ -3,7 +3,7 @@ vcpkg_buildpath_length_warning(37)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO InsightSoftwareConsortium/ITK
-    REF 95800fd4d4b08678a6c0ebb63eb242893025b660 #5.2.1
+    REF v5.3-rc02
     SHA512 fe703bc6ed681cb9983d7d6e21c8ffa7650337e470c09a7241de58a463c23e315516b1a81a18c14f682706056a0ec66932b63d2e24945bdcea03169bc1122bb2
     HEAD_REF master
     PATCHES
@@ -15,6 +15,7 @@ vcpkg_from_github(
         opencl.patch
         use-the-lrintf-intrinsic.patch
         dont-build-gtest.patch
+        cxx17.patch # See https://github.com/InsightSoftwareConsortium/ITK/issues/4213#issuecomment-1722547712
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS

--- a/ports/itk/vcpkg.json
+++ b/ports/itk/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "itk",
-  "version": "5.2.1",
-  "port-version": 5,
+  "version": "5.3-rc02",
+  "port-version": 6,
   "description": "Insight Segmentation and Registration Toolkit (ITK) is used for image processing and analysis.",
   "homepage": "https://github.com/InsightSoftwareConsortium/ITK",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3477,8 +3477,8 @@
       "port-version": 0
     },
     "itk": {
-      "baseline": "5.2.1",
-      "port-version": 5
+      "baseline": "5.3-rc02",
+      "port-version": 6
     },
     "itpp": {
       "baseline": "4.3.1",

--- a/versions/i-/itk.json
+++ b/versions/i-/itk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9da4bc8949edd65bc3e0afa8e694480e330f5f40",
+      "version": "5.3-rc02",
+      "port-version": 6
+    },
+    {
       "git-tree": "33def29a955f79fae7497d8be276decf175a9881",
       "version": "5.2.1",
       "port-version": 5

